config/config_test.go

```diff
@@ -34,6 +35,91 @@ func TestT(t *testing.T) {
 	TestingT(t)
 }
 
+func (s *testConfigSuite) TestNullableBoolUnmashal(c *C) {
+	var nb = nullableBool{false, false}
+	data, err := json.Marshal(nb)
+	c.Assert(err, IsNil)
+	err = json.Unmarshal(data, &nb)
+	c.Assert(err, IsNil)
+	c.Assert(nb, Equals, nbUnset)
+
+	nb = nullableBool{true, false}
+	data, err = json.Marshal(nb)
+	c.Assert(err, IsNil)
+	err = json.Unmarshal(data, &nb)
+	c.Assert(err, IsNil)
+	c.Assert(nb, Equals, nbFalse)
+
+	nb = nullableBool{true, true}
+	data, err = json.Marshal(nb)
+	c.Assert(err, IsNil)
+	err = json.Unmarshal(data, &nb)
+	c.Assert(err, IsNil)
+	c.Assert(nb, Equals, nbTrue)
+}
+
+func (s *testConfigSuite) TestLogConfig(c *C) {
+	var conf Config
+	configFile := "log_config.toml"
+	_, localFile, _, _ := runtime.Caller(0)
+	configFile = path.Join(path.Dir(localFile), configFile)
+
+	f, err := os.Create(configFile)
+	c.Assert(err, IsNil)
+
+	var testLoad = func(confStr string, expectedEnableErrorStack, expectedDisableErrorStack, expectedEnableTimestamp, expectedDisableTimestamp nullableBool, resultedDisableTimestamp, resultedDisableErrorVerbose bool, valid Checker) {
+		conf = defaultConf
+		_, err = f.WriteString(confStr)
+		c.Assert(err, IsNil)
+		c.Assert(f.Sync(), IsNil)
```

Comment:
This line is useless.

Reply:
You mean
```
c.Assert(f.Sync(), IsNil)
```
?

I could remove it to make you happy, but it is also borrowed from previous code.

```
f, err := os.Create(configFile)
	c.Assert(err, IsNil)

	// Make sure the server refuses to start if there's an unrecognized configuration option
	_, err = f.WriteString(`
unrecognized-option-test = true
`)
	c.Assert(err, IsNil)
	c.Assert(f.Sync(), IsNil)

	c.Assert(conf.Load(configFile), ErrorMatches, "(?:.|\n)*unknown configuration option(?:.|\n)*")

```

Reply:
You can change the previous code. :)